src/utils/common.ts

```diff
@@ -89,3 +89,13 @@ export const buildUrlOptions = (options: { [key: string]: string }): string => {
   });
   return optionStr;
 };
+
+export const onCopy = async (valueToCopy: string, onSuccess?: () => void) => {
```

Comment:
since it's no longer an event I would say we rename it to action based such as `copyToClipboard` or similar

Reply:
okay sure!